include/openssl/x509.h.in

```diff
@@ -1025,6 +1025,9 @@ int EVP_PKEY_add1_attr_by_txt(EVP_PKEY *key,
                               const char *attrname, int type,
                               const unsigned char *bytes, int len);
 
+STACK_OF(X509) *OSSL_build_cert_chain(OSSL_LIB_CTX *libctx, const char *propq,
```

Comment:
Do we really want this to be a public API?

Reply:
So far it is not strictly needed but may be of more general interest to certain applications.
For instance, mbedTLS export such a function.
So far the chain built by OpenSSL can be extracted only in conjunction with chain verification,
which may be more than needed/wanted for some use cases.

Reply:
I've meanwhile taken the addition of a generally useful chain building function to a separate PR: #14128.